src/main.rs

```diff
@@ -39,6 +39,7 @@ fn main() {
 
 	vm::run(bin_path, debug);
 
-	let end = PreciseTime::now();
-	println!("Took {}", start.to(end));
+	let end = Instant::now();
+	let elapsed = start - end;
+	println!("Took {}:{}:{}", elapsed.whole_minutes(), elapsed.whole_seconds(), elapsed.whole_milliseconds());
```

Comment:
This is the main area that probably would have changed behavior. Not sure what `std::Fmt` would have done before but it's not implemented on `Duration` anymore. So I winged it (wanged it?).

Reply:
Wang'n it worked!